### ISSUE TITLE
Downloading new beta from OpenAudible App doesn't work

### ISSUE THREAD

**Issue by RandmTask** [openaudible/openaudible#729]
Hi,

When I try to download the latest version (beta) in the OpenAudible **app** when I have a notification that there is a new beta, it opens up openaudible.org/beta in the **in-app** browser -> I choose the beta to download -> it opens the save dialog box, when I hit Save it does not download however.

I can download OpenAudible from openaudible.org/beta just fine from Safari/Chrome however.

Big Sur 11.4 with 3.1.2.1 beta trying to download 3.1.2.2 beta (Ive never actually had the inapt download work over multiple versions however)

You can see the blue progress bar but it never moves and no files are downloaded (Have tried downloading Windows and Linux versions also)
![Screen Shot 2021-09-26 at 8 21 02 am](https://user-images.githubusercontent.com/7912017/134793764-9479b121-2b91-4d44-9d56-93fa9301c5bf.png)

tl;dr Cannot download from the in-app browser, only from Safari or Chrome.

Thanks

**Comment by openaudible** [comment]
Weird. Does downloading from: 
https://github.com/openaudible/openaudible/releases/latest
work?
Any firewall or anti-virus on your system?

**Comment by RandmTask** [comment]
Yep, strange!

Works fine from that GitHub link though (in Safari). 

Have tried with and without VPN with, and without a firewall active. No antivirus either

**Comment by openaudible** [comment]
Confirmed. Thank you for the report. No easy fix. Will update the browser library we use. No guarantees. Closing but will keep thinking about why.
